easyenergy/models.py

```diff
@@ -201,6 +201,28 @@ def timestamp_return_prices(self) -> list[dict[str, float | datetime]]:
         """
         return self.generate_timestamp_list(self.return_prices)
 
+    @property
+    def hours_priced_equal_or_better_usage(self) -> int:
+        """Return the number of hours with the current price or better for usage.
+
+        Returns
+        -------
+            The number of hours with the current price or better for usage.
```

Comment:
```suggestion
            The number of hours with the current price for usage.
```

Reply:
That wouldn't be quite correct.  It's counting the number of hours that are the current price or below in case of usage.

Reply:
But also `or better` says nothing about what the code does 😉

Reply:
Updated to reflect what the code does.

Reply:
Resolved in [66f426b](https://github.com/klaasnicolaas/python-easyenergy/pull/55/commits/66f426b2545ca4da5fe4319d80a7cb3e133b3a20)